modules/simple_forms_api/app/services/simple_forms_api/s3/utils.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'reports/uploader'
+
+module SimpleFormsApi
+  module S3
+    class Utils
+      private
+
+      def assign_instance_variables(defaults)
```

Comment:
As noted above, I'm not entirely sold on the utility of this method.

Reply:
See my above response, this isn't a hill I'm ready to die on but I do like how it cleans up the code.